### PR TITLE
Tutorial fixes

### PR DESCRIPTION
**Pull request by huffman**
The tutorial changes do not need to be QA'd, the majority of these changes have been in our internal tests for awhile, and the small changes I have made I have tested myself. What should be tested is that the vive controller display is working as expected.

To test:

1. Run `HMD.requestShowHandControllers();` and ensure that the hand controllers show up, and look correct
1. Ensure that running `Messages.sendLocalMessage('Controller-Set-Part-Layer', JSON.stringify({ trigger: "highlight" }));` highlights the controllers trigger yellow
1. Run `HMD.requestHideHandControllers();` and ensure that the hand controllers show up, and look correct

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 69e7dababc58634a00acc08fed8050f65a26797a available at: https://gist.github.com/83d952fc8e592866c397a64ac16a57a4<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8939-69e7dab.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8939-69e7dab.dmg

**Comment by thoys** [comment]
## QA testing report

0. The tutorial works fine
1. `HMD.requestShowHandControllers();` shows the controllers ☑️ 
2. `Messages.sendLocalMessage('Controller-Set-Part-Layer', JSON.stringify({ trigger: "highlight" }));` shows the trigger in white (Possibly the white texture is missing) 👎 
3. ` HMD.requestHideHandControllers()` hides the controllers ☑️ 


QA Fail  (Highlighted button is white, not yellow)

**Comment by hifi-gustavo** [comment]
Build console output for commit: d933155c71f354b278c5e1fb8cc272aa200288fb available at: https://gist.github.com/83d952fc8e592866c397a64ac16a57a4<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8939-d933155.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8939-d933155.dmg

**Comment by thoys** [comment]
## QA Testing report (update)

Retested all, including tutorial, works well, the trigger is now highlighter in yellow.

QA Pass

File: scripts/system/controllers/viveControllerConfiguration.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ var TIP_TEXTURE_BASE_URL = BASE_URL + "meshes/controller/vive_tips.fbm/";
 
 var viveModelURL = BASE_URL + "meshes/controller/vive_body.fbx";
 var viveTipsModelURL = BASE_URL + "meshes/controller/vive_tips.fbx";
+var viveTriggerModelURL = "meshes/controller/vive_trigger.fbx"
```
Comment (birarda) [inline scripts/system/controllers/viveControllerConfiguration.js]: do you need BASE_URL here?

Comment (birarda) [inline scripts/system/controllers/viveControllerConfiguration.js]: Ah, you use it below

Comment (birarda) [inline scripts/system/controllers/viveControllerConfiguration.js]: Would it be cleaner to use it though?

**Pull request merged**